File: iOS/LearnObjective-C/L09NSDateDemo/L09NSDateDemo/ViewController.m
//
//  ViewController.m
//  L09NSDateDemo
//
//  Created by mengai on 16/8/10.
//  Copyright © 2016年 Meng. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UILabel *currentDateLabel;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSDate *date = [[NSDate alloc] init];
    self.currentDateLabel.text = [date descriptionWithLocale:@"shanghai"];
    
    NSTimeInterval timei = [date timeIntervalSinceReferenceDate];
    NSLog(@"timei = %f",timei);
    
    
    
    NSLog(@"distantFuture = %@",[NSDate distantFuture]);
    NSLog(@"distantPast = %@",[NSDate distantPast]);
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: iOS/LearnSwift/Swift3.0/Grammar/L09NSDate/L09NSDate/ViewController.swift
//
//  ViewController.swift
//  L09NSDate
//
//  Created by mengai on 16/8/10.
//  Copyright © 2016年 Meng. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var currentDateLabel: UILabel!

    override func viewDidLoad() {
        super.viewDidLoad()
        
        
        
        pring(text: "哈哈");
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    func pring(text : String) -> Void {
        NSLog("%@", text);
    }
    

}

